### DISCUSSION TITLE
LA104 - WebUSB signal analyser running in chrome but not connect to the device.

### DISCUSSION THREAD

**Discussion by Azanot** [gabonator/LA104#63] [Q&A]
Hi Gabriel, congratulations for your work. Hope you can help me.. I can't use the web interface for RF analysis. 
Chrome recognizes the usb device with a popup and clicking on the popup open "CC1101 analyzer" web page but staying still on "Connecting" If I click on the connect button i can choose the peripheral "LA104" from the list but not receiving any signal, only simulator works. 
Thanks  

Browser: Chrome V.91.0.4472.114 (Build ufficiale) (a 64 bit)
SO: Windows 7 Pro - 64 bit
Device: LA104 - V.3.61D

![01](https://user-images.githubusercontent.com/86295884/122912905-1e5ea000-d359-11eb-86c1-48c1dbd82e8c.JPG)
![02](https://user-images.githubusercontent.com/86295884/122912911-1f8fcd00-d359-11eb-8cad-97f48cc77e9a.JPG)
![03](https://user-images.githubusercontent.com/86295884/122912914-20c0fa00-d359-11eb-939e-419d8784a9a1.JPG)
![04](https://user-images.githubusercontent.com/86295884/122912915-21599080-d359-11eb-9eb5-8627dc567fd4.JPG)
![05](https://user-images.githubusercontent.com/86295884/122916993-d130fd00-d35d-11eb-8e26-1d623476d1c5.JPG)

**Comment by gabonator** [comment]
Could you take a look at this issue? https://github.com/gabonator/LA104/issues/51 There is some workaround through developer console described, maybe it will help
